modules/services/podman-linux/default.nix

```diff
@@ -1,14 +1,13 @@
 { pkgs, lib, ... }:
 
-with lib;
-
 {
+  meta.maintainers = with lib.hm.maintainers; [ bamhm182 n-hass ];
+
   imports =
     [ ./containers.nix ./install-quadlet.nix ./networks.nix ./services.nix ];
 
-  config = mkIf pkgs.stdenv.isLinux {
```

Comment:
@rycee removing this has caused eval to fail on Darwin as it hits the platform assertion. If not having this top-level `mkIf`, what is the preferred way to do the platform guard?

Reply:
I think the best solution is to add an option `services.podman.enable` that defaults to `false`. This will guard the inner workings of the module and reduce the amount of code that needs to be evaluated.

It also allows a way to fully disable the module with a single option. I've added this option in a new commit, hopefully that resolves the CI error.